### DISCUSSION TITLE
Component already initialized and Component not found

### DISCUSSION THREAD

**Discussion by mq1lv** [livewire/livewire#8760] [1. Help]
Hi all! Have an issue with livewire components.

So I have component "Products". There is Search input (wire:model="search") and there is a table, which loops over db records. Each row has livewire component - pretty simple just to update value in db on enter key. 

So when I load Products component, I see products list , no errors.
When I enter some search term - it searches (db search) and I see only found products in the list. No errors.
When I clear Search input, I see all products, but I got error: Component already initialized.
When I enter any other search term - I got error: Component not found

There is wire:key on loop elemnts. Each livewire component in row has :key. All keys have prefixed.

Where could be error?

Thanks!

**Comment by workualemu** [comment]
Any solution to this issue? I currently face same issue.
